packages/experimental/Button/src/ButtonColorTokens.ts

```diff
@@ -17,13 +17,13 @@ export const defaultButtonColorTokens: TokenSettings<ButtonTokens, Theme> = (t:
     hovered: {
       backgroundColor: t.colors.defaultHoveredBackground,
       color: t.colors.defaultHoveredContent,
-      borderColor: t.colors.defaultHoveredBorder,
+      borderColor: t.colors.buttonBorder,
       iconColor: t.colors.defaultHoveredIcon,
     },
     pressed: {
       backgroundColor: t.colors.defaultPressedBackground,
       color: t.colors.defaultPressedContent,
-      borderColor: t.colors.defaultPressedBorder,
+      borderColor: t.colors.buttonBorder,
```

Comment:
Hmm, let me take a look at those tokens.

Reply:
Good point, I'm reverting these changes and updating the default border tokens to use transparentStroke.

Aside, it may be a good idea to have a separate Button colors tokens file for macOS.